internal/storage/integration/integration.go

```diff
@@ -193,12 +195,13 @@ func (s *StorageIntegration) testGetServices(t *testing.T) {
 					StartTimeMin: time.Now().Add(-2 * time.Hour),
 					StartTimeMax: time.Now(),
 				})
-				traces, err := v1adapter.V1TracesFromSeq2(iterTraces)
+				traces, err := jiter.FlattenWithErrors(iterTraces)
```

Comment:
what is the point of this change? How is this better than calling `v1adapter.V1TracesFromSeq2`?